app/uk/gov/hmrc/customs/emailfrontend/config/ErrorHandler.scala

```diff
@@ -25,7 +25,7 @@ import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
 import uk.gov.hmrc.customs.emailfrontend.views.html.partials.error_template
 
 @Singleton
-class ErrorHandler @Inject()(val messagesApi: MessagesApi, implicit val appConfig: AppConfig, errorView: error_template) extends FrontendErrorHandler {
+class ErrorHandler @Inject()(val messagesApi: MessagesApi, errorView: error_template) extends FrontendErrorHandler {
```

Comment:
do we need this val here?

Reply:
I removed everywhere override keyword from all other places now since we realised that MessagesApi is abstract for I18nSupport therefore does not require to be overwritten.